content/source/docs/enterprise/admin/update-tfe-license.html.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: "enterprise"
+page_title: "Updating Terraform Enterprise License - Application Administration - Terraform Enterprise"
+---
+## Introduction
+
+Terraform Enterprise requires a license in order to maintain normal operations. These licenses have an expiration date, and must be updated once expired. This article provides the guidelines for updating licenses in Terraform Enterprise.
+
+## Finding the expiration date
+
+There are two methods to find the expiration date from the existing installation of Terraform Enterprise; the Replicated console UI or the Replicated command line interface.
+
+### Replicated Console
+
+1.  Navigate to the Replicated console which is located at `https://<<TFE_HOSTNAME>>:8800` and click on the gear icon on the top-right corner.
+
+    ![View License](./images/view-license.png)
+
+2.  Select the **View License** menu and the details of the existing license will be displayed on the screen with the expiration date.
+
+    ![License Expiration](./images/license-expiration.png)
+
+### Replicated Command Line Interface
+
+1.  Connect to the Terraform Enterprise host machine using SSH.
+
+2.  Use the `replicatedctl license inspect` command to inspect the license information. The `ExpirationTime` value in the output indicates the expiration date/time in UTC.
+
+        $ replicatedctl license inspect
+        [
+        {
+           "ID": "670bd320b95245325d80be703330f9b3",
+           "Assignee": "Test Customer Name",
+           "LegacyChannelName": "Stable",
+           "ChannelID": "",
+           "Channels": [],
+           "ExpirationTime": "2021-06-30T00:00:00Z",
+           "ExpirationPolicy": "ignore",
+           "IsExpired": false,
+           "IsActivationRequired": false,
+           "ActivationEmail": "",
+           "Fields": [
+               {
+                   "FieldName": "max_hosts",
+                   "FieldTitle": "Maximum Number of Hosts",
+                   "FieldType": "Integer",
+                   "Value": "9999"
+               },
+               {
+                   "FieldName": "min_hosts",
+                   "FieldTitle": "Minimum Number of Hosts",
+                   "FieldType": "Integer",
+                   "Value": "1"
+               }
+           ]
+        }
+        ]
+
```

Comment:
We should wrap this in a code fence

````suggestion
    ```shell
        $ replicatedctl license inspect
        [
        {
           "ID": "670bd320b95245325d80be703330f9b3",
           "Assignee": "Test Customer Name",
           "LegacyChannelName": "Stable",
           "ChannelID": "",
           "Channels": [],
           "ExpirationTime": "2021-06-30T00:00:00Z",
           "ExpirationPolicy": "ignore",
           "IsExpired": false,
           "IsActivationRequired": false,
           "ActivationEmail": "",
           "Fields": [
               {
                   "FieldName": "max_hosts",
                   "FieldTitle": "Maximum Number of Hosts",
                   "FieldType": "Integer",
                   "Value": "9999"
               },
               {
                   "FieldName": "min_hosts",
                   "FieldTitle": "Minimum Number of Hosts",
                   "FieldType": "Integer",
                   "Value": "1"
               }
           ]
        }
        ]
    ```

````